src/styles/cards.css

```diff
@@ -0,0 +1,80 @@
+.cards {
+  box-sizing: border-box;
+  color: #060b35;
+  width: 200px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  border: 1px solid #f3f3f3;
+
+  &__foto {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
```

Comment:
The BEM element class should be written as `.cards__foto` instead of using the `&` which is a SASS/SCSS syntax. Since this is a CSS file, you should not use SASS/SCSS features.